src/alcotest/core.ml

```diff
@@ -345,7 +345,9 @@ module Make (M : Monad.S) = struct
     let open Suite in
     let test = suite.fn in
     let pp_event = pp_event t in
+    M.return () >>= fun () ->
     pp_event Fmt.stdout (`Start suite.path);
+    Fmt.(flush stdout) () (* Show event before any test stderr *);
```

Comment:
why not printing stderr here too?

Reply:
There shouldn't be any pending `stderr` here: the `stderr` of each test case is flushed as soon as it finishes and all we've done since then is print events to `stdout`.